docs/content/docs/guide/incoming_webhook.md

```diff
@@ -14,10 +14,9 @@ specify a reference Secret which will be used as a shared secret and the
 branches targetted by the incoming webhook.
 
 {{< hint danger >}}
-you will need to have a git_provider spec to specify a token when using the
-github-apps method the same way we are doing for github-webhook method. Refer to
-the [github webhook documentation](/docs/install/github_webhook) for how to set
-this up.
+you will need to have a `git_provider` spec to specify a token when using the
+github-webhook method. Refer to the [github webhook documentation](/docs/install/github_webhook)
```

Comment:
a non `github app` ? because it's not only github-webhook

Reply:
Updated doc